### ISSUE TITLE
Forerunner® 645 Music -  non supported device

### ISSUE THREAD

**Issue by stephf26** [adamjakab/HiiTiT#17]
Hi Adam
it seems you have the best iHiit app, it will be great if you can make it available for the Forerunner® 645 Music.

thanks a lot

Ps: I wrote here, because I can not write a review on Garmin if I do not have the app

Stephanie

**Comment by adamjakab** [comment]
Hi @stephf26,

I actually prefer that you wrote your request here because it is much more trackable for me than the IQ reviews. 

I will keep you updated on the status of this request. Having said that I think it will take a while. Before I developed iHIIT I used to use this one [Work-It-Out Fitness Interval Training](https://apps.garmin.com/en-US/apps/8a7570e8-752a-420a-b0db-83376c4e2f53) which is compatible with your device. I decided to make a new one because I was missing some key features from it (I cannot recall which). Until then give it a try.
